eng/scripts/typespec/Build-Emitter.ps1

```diff
@@ -1,83 +1,76 @@
 #Requires -Version 7.0
 
 param(
-    [string] $BuildNumber,
     [string] $OutputDirectory,
-    [switch] $Prerelease,
-    [switch] $TargetNpmJsFeed
+    [switch] $TargetNpmJsFeed,
```

Comment:
We should consider modifying the trigger for eng pipeline to not run when this folder is changed since these scripts only affect the emitter packages, we should also make sure the trigger for both emitters run if this folder is changed.

Reply:
Which pipeline would this be?

Reply:
The Core pipeline already does this - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/ci.yml#L17